diem-unified-charter.md

```diff
@@ -26,16 +26,16 @@ Which attributes an emblem contains, and how a digital emblem is secured and pre
 
 # Initial Scope
 
-The DIEM WG will ensure maximum reuse of work for emblem representation and validation, facilitate re-use of existing protocols and capabilities, and ensure that existing standards are leveraged appropriately.
-A discovery mechanism for the initial work will only be specified by this group after the initial emblem validation procedure is completed (see the Deliverables section below).
+The working group will initially develop a discovery mechanism using the DNS, and digital emblems that identify their bearer using domain names or implicitly.
```

Comment:
Does discovery need to be limited to DNS?  Or could it be URL-based in general, with one of the options being DNS?